extensions/mssql/src/main.ts

```diff
@@ -137,35 +137,47 @@ export async function activate(context: vscode.ExtensionContext): Promise<MssqlE
+
+		});
 		if (endpointsArray.length > 0) {
-			const managementProxyEp = endpointsArray.find(e => e.serviceName === 'management-proxy');
+			const managementProxyEp = endpointsArray.find(e => e.serviceName === 'management-proxy' || e.serviceName === 'mgmtproxy');
 			if (managementProxyEp) {
-				endpointsArray.push(getCustomEndpoint(managementProxyEp, 'Grafana Dashboard', '/grafana'));
-				endpointsArray.push(getCustomEndpoint(managementProxyEp, 'Kibana Dashboard', '/kibana'));
+				endpointsArray.push(getCustomEndpoint(managementProxyEp, 'Metrics Dashboard', '/grafana'));
+				endpointsArray.push(getCustomEndpoint(managementProxyEp, 'Log Search Dashboard', '/kibana'));
 			}
 
 			const gatewayEp = endpointsArray.find(e => e.serviceName === 'gateway');
 			if (gatewayEp) {
-				endpointsArray.push(getCustomEndpoint(gatewayEp, 'Spark History', '/gateway/default/sparkhistory'));
-				endpointsArray.push(getCustomEndpoint(gatewayEp, 'Yarn History', '/gateway/default/yarn'));
+				endpointsArray.push(getCustomEndpoint(gatewayEp, 'Spark Job Monitoring', '/gateway/default/sparkhistory'));
+				endpointsArray.push(getCustomEndpoint(gatewayEp, 'Spark Resource Management', '/gateway/default/yarn'));
 			}
 
 			const container = view.modelBuilder.flexContainer().withLayout({ flexFlow: 'column', width: '100%', height: '100%', alignItems: 'left' }).component();
 			endpointsArray.forEach(endpointInfo => {
 				const endPointRow = view.modelBuilder.flexContainer().withLayout({ flexFlow: 'row' }).component();
-				const nameCell = view.modelBuilder.text().withProperties<azdata.TextComponentProperties>({ value: endpointInfo.serviceName }).component();
-				endPointRow.addItem(nameCell, { CSSStyles: { 'width': '30%', 'font-weight': '600' } });
+				const nameCell = view.modelBuilder.text().withProperties<azdata.TextComponentProperties>({ value: getFriendlyEndpointNames(endpointInfo.serviceName) }).component();
+				endPointRow.addItem(nameCell, { CSSStyles: { 'width': '35%', 'font-weight': '600', 'user-select': 'text' } });
 				if (endpointInfo.isHyperlink) {
 					const linkCell = view.modelBuilder.hyperlink().withProperties<azdata.HyperlinkComponentProperties>({ label: endpointInfo.hyperlink, url: endpointInfo.hyperlink, position: '' }).component();
-					endPointRow.addItem(linkCell, { CSSStyles: { 'width': '70%', 'color': 'blue', 'text-decoration': 'underline', 'padding-top': '10px' } });
+					endPointRow.addItem(linkCell, { CSSStyles: { 'width': '60%', 'color': 'blue', 'text-decoration': 'underline', 'padding-top': '10px' } });
 				}
 				else {
 					const endpointCell = view.modelBuilder.text().withProperties<azdata.TextComponentProperties>({ value: endpointInfo.ipAddress + ':' + endpointInfo.port }).component();
-					endPointRow.addItem(endpointCell, { CSSStyles: { 'width': '70%' } });
+					endPointRow.addItem(endpointCell, { CSSStyles: { 'width': '60%', 'user-select': 'text' } });
```

Comment:
Could we just make the entire table `user-select`able?

Reply:
added the user-select to the container.